dev/com.ibm.ws.channelfw/src/com/ibm/ws/bytebuffer/internal/WsByteBufferImpl.java

```diff
@@ -73,8 +73,8 @@
     protected int quickBufferAction = NOT_ACTIVATED;
 
     // min/max are inclusive, so threshold will be this number + 1
-    private int GET_THRESHOLD = 1023; // minimum amount to copy from Direct to NonDirect
-    private int PUT_THRESHOLD = 2047; // maximum number of bytes between non-contiguous puts that will be bridged.
+    private final int GET_THRESHOLD = 1023; // minimum amount to copy from Direct to NonDirect
```

Comment:
Making these static (in addition to final) could result in better JIT optimization (folding away of the loads from memory to be constants instead).

Reply:
Eclipse added the final automatically as part of the automatic class formatting.  It was not an explicit change to the code.  I have updated the code now though to use static.